src/ooxml/java/org/apache/poi/xssf/usermodel/helpers/XSSFColumnShifter.java

```diff
@@ -0,0 +1,170 @@
+package org.apache.poi.xssf.usermodel.helpers;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+import org.apache.poi.ss.ITestDataProvider;
```

Comment:
`ITestDataProvider` is part of the unit test suite and not included in any of the poi jars. This class will be unrunnable outside of a development environment.